checkov/common/output/report.py

```diff
@@ -593,6 +595,13 @@ def handle_skipped_checks(
                 report.failed_checks.remove(record)
         return report
 
+    @staticmethod
+    def get_plan_resource_raw_id(resource_id: str) -> str:
+        resource_raw_id = ".".join(resource_id.split(".")[-2:])
```

Comment:
didn't you did something just like that somewhere else?

Reply:
a doc string with an example would be great 🙂

Reply:
no